languages/csharp/exercises/concept/properties/.docs/after.md

```diff
@@ -0,0 +1,73 @@
+The two main types of property are 
+
+1. auto-implemented properties where the `get` and `set` accessors have no body.
+They may or may not be explicitly initialized.  For example:
+``` csharp
```

Comment:
This list is not correctly formatted, and not contiguous . In order to fix it (and make it accessible), indent the code blocks (four spaces). You can _test_ if a list is contiguous by using the value `1.` or `a.` for _all_ list items. Markdown will correctly turn it into a monotonically increasing numbered or lettered list if it's correctly formatted.

````markdown
1. Auto-implemented properties where the `get` and `set` accessors have no body. They may or may not be explicitly initialized.  For example:
  
    ``` csharp
    public int MyProperty {get; set;} = 42;
    ```

1. Those where the accessors evaluate expressions and execute statements.  The code can be as simple as returning or assigning a backing field.  For example:

    ``` csharp
    private int myField;
    public int MyProperty
    {
      get { return myField; }
      set { myField = value; }
    }
    ```
1. hello
```` 

becomes

1. Auto-implemented properties where the `get` and `set` accessors have no body. They may or may not be explicitly initialized.  For example:
  
    ``` csharp
    public int MyProperty {get; set;} = 42;
    ```

1. Those where the accessors evaluate expressions and execute statements.  The code can be as simple as returning or assigning a backing field.  For example:

    ``` csharp
    private int myField;
    public int MyProperty
    {
      get { return myField; }
      set { myField = value; }
    }
    ```

1. hello

Reply:
Done